internal/pkg/keystores/validate_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2022 Nethermind
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package keystores
+
+import (
+	"embed"
+	"errors"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed testdata
+var testdata embed.FS
+
+func TestValidateKeystoreDir(t *testing.T) {
```

Comment:
Add the following test cases please:

- [ ] Valid keystore folder but without deposit_data
- [ ] Valid keystore folder but without password file
- [ ] Valid keystore folder but without validator_keys folder
- [ ] Valid keystore folder but without keystore.json files inside validator_keys folder

Reply:
Done https://github.com/NethermindEth/sedge/pull/228/commits/7506c3a7f94388c054276e895ea19cfbe46d8fda